src/lib.rs

```diff
@@ -49,13 +46,13 @@ lazy_static::lazy_static! {
     pub static ref ECMULT_GEN_CONTEXT: Box<ECMultGenContext> = ECMultGenContext::new_boxed();
 }
 
-#[cfg(feature = "static-context")]
+#[cfg(all(feature = "static-context", not(feature = "lazy-static-context")))]
 /// A static ECMult context.
 // Correct `pre_g` values are fed into `ECMultContext::new_from_raw`, generated by build script.
```

Comment:
```suggestion
/// A static ECMult context. Requires that the `static-context` to be enabled and that `lazy-static-context` is **not** enabled.
/// Correct `pre_g` values are fed into `ECMultContext::new_from_raw`, generated by build script.
```

Reply:
`ECMULT_CONTEXT` is defined either here or through `lazy_static`. It is always available either with `static-context` or `lazy-static-context`.

The second line is related to implementation details of how this is generated, so it doesn't really fit the docs.